### ISSUE TITLE
Sometimes, very slow (~10 seconds?) on the 1st time to unmute self in a conference call with 6+ participants

### ISSUE THREAD

**Issue by stephendonner** [jitsi/jitsi-meet#9736]
### Description:

Sometimes, very slow (~10 seconds?) on the 1st time to unmute self in a conference call with 6+ participants.

### Steps to reproduce:

1. launch a Brave Talk (powered by JaaS, hence filing here) call via `https://talk.brave.com/` (note: you will need to use the latest release version of Brave)
2. make sure to join while muted
3. wait a few minutes/invite a few more people to the call (this typically happens for us in calls with 6+ participants, all of whom join muted)
4. click on the `microphone` icon to unmute yourself
5. notice the button either takes a long time/many presses to unmute, --or-- it appears to unmute successfully via the icon state, but is actually still muted

@btlechowski: please correct/augment the above steps; thanks!  @amirsaber I believe you have a 100% repro case too?

### Expected behavior:

Mute/unmute should be as instantaneous as possible.

### Actual behavior:

It can take up to ~20 seconds to successfully unmute yourself.

Sorry, we don't yet have logs, but will attach them here the next time we encounter this.

### Client information:

- Browser / app version: Brave, latest release
- Operating System: Windows, Mac, iOS

**Comment by kjozwiak** [comment]
+1 as I've ran into the above on several occasions during our QA standup meetings. Unfortunately I don't have a 100% reproducible case but I've definitely experienced this a few times. Basically what @stephendonner mentioned. There's two cases:

#### `Case #1`

* Join a meeting and wait till a few more people join
* click on the unmute button and you'll notice that it didn't do anything

In this particular case, it takes several clicks to finally unmute yourself.

#### `Case #2`

 * Join a meeting and wait till a few more people join
* click on the unmute button and the UI will correctly update/indicate that you're unmuted
* start speaking but no one will hear you (usually people in the meeting will mention they can't hear you)
* click mute/unmute again and eventually people in the meeting will be able to hear you

The above two cases only happened a handful of times but definitely occur.

**Comment by saghul** [comment]
If you join unmuted, then mute, then other participants join, then you unmute, do you observe this behavior?

**Comment by stephendonner** [comment]
> If you join unmuted, then mute, then other participants join, then you unmute, do you observe this behavior?

No, in my limited testing just now, and experience, this works; @amirsaber does that match yours, too?

**Comment by amirsaber** [comment]
Yes it matches mine.

**Comment by saghul** [comment]
I think the difference is we create the track later, lazily. @jallamsetty1 any idea why it might take that long?

**Comment by saghul** [comment]
@jallamsetty1 Did you manage to take a look?

**Comment by stephendonner** [comment]
@jallamsetty1 apologies for the delay; here's a quick run on my Brave on iOS client.  Please let me know if you need more runs or additional info; thanks!

![Uploading IMG_0099.PNG…]()

**Comment by jallamsetty1** [comment]
> @jallamsetty1 apologies for the delay; here's a quick run on my Brave on iOS client. Please let me know if you need more runs or additional info; thanks!
> 
> ![IMG_0099](https://user-images.githubusercontent.com/387249/131537781-04a4d2e8-e6b1-4dfb-8fba-224e76496ebd.PNG)

Thanks @stephendonner, are you only seeing this on iOS? Not being able to unmute on iOS is a different issue and we recently addressed it in this [PR](https://github.com/jitsi/jitsi-meet/pull/9837). We should be able to push the fix to prod in the next release. Are you seeing delays on desktop as well ?

**Comment by stephendonner** [comment]
Hi @jallamsetty1 - I don't have problems unmuting on desktop, but I know @btlechowski and @amirsaber do; can you both please run the test at https://webrtc.github.io/samples/src/content/peerconnection/negotiate-timing/?  I think it's just click `Start`, then click `Call`, and then `Renegotiate` - click this last button a good number of times, to get an average/median latency.

**Comment by amirsaber** [comment]
@stephendonner no sound until unmute issue that I am facing is in fact on iOS only. 
Desktop results:
![image](https://user-images.githubusercontent.com/4715448/131586962-dcc8ad65-2213-48ce-b9f5-a7392b8f544a.png)
iOS results:
![C74752D2-698B-40B4-9984-A37607F2CE71_1_105_c](https://user-images.githubusercontent.com/4715448/131587077-d9a9f3b4-300e-47e9-a99d-90d9e8f9fe88.jpeg)

**Comment by jallamsetty1** [comment]
@stephendonner, I was out last week and missed this message. Should we keep this issue open until the next prod release and close it if no one else is able to reproduce this issue ?

**Comment by stephendonner** [comment]
> @stephendonner, I was out last week and missed this message. Should we keep this issue open until the next prod release and close it if no one else is able to reproduce this issue ?

@jallamsetty1 sounds like a good plan to me; thanks!
